pkg/antctl/raw/multicluster/create/access_token.go

```diff
@@ -0,0 +1,188 @@
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"antrea.io/antrea/pkg/antctl/raw"
+	multiclusterscheme "antrea.io/antrea/pkg/antctl/raw/multicluster/scheme"
+)
+
+type accessTokenOptions struct {
+	namespace      string
+	serviceAccount string
+	roleBinding    string
+}
+
+var accessTokenOpts *accessTokenOptions
+
+var accessTokenExamples = strings.Trim(`
+Create an access token in a leader cluster for one or more member clusters. If the ServiceAccount or RoleBinding does not exist, antctl will create one
+$ antctl create accesstoken -n <NAMESPACE> --service-account <SERVICE_ACCOUNT> --role-binding <ROLE_BINDING>
+`, "\n")
+
+func (o *accessTokenOptions) validateAndComplete() error {
+	if o.namespace == "" {
+		return fmt.Errorf("the Namespace cannot be empty")
+	}
+	if o.serviceAccount == "" {
+		return fmt.Errorf("the ServiceAccount cannot be empty")
+	}
+	if o.roleBinding == "" {
+		return fmt.Errorf("the RoleBinding cannot be empty")
+	}
+	return nil
+}
+
+func NewAccessTokenCmd() *cobra.Command {
+	command := &cobra.Command{
+		Use:     "accesstoken",
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "Create an accesstoken in a leader cluster",
+		Long:    "Create an accesstoken in a leader cluster",
+		Example: accessTokenExamples,
+		RunE:    accessTokenRunE,
+	}
+
+	o := &accessTokenOptions{}
+	accessTokenOpts = o
+	command.Flags().StringVarP(&o.namespace, "namespace", "n", "", "Namespace of the ClusterClaim")
+	command.Flags().StringVarP(&o.serviceAccount, "service-account", "", "", "ServiceAccount of the access token")
+	command.Flags().StringVarP(&o.roleBinding, "role-binding", "", "", "RoleBinding of the ServiceAccount")
+
+	return command
+}
+
+func accessTokenRunE(cmd *cobra.Command, args []string) error {
+	if err := accessTokenOpts.validateAndComplete(); err != nil {
+		return err
+	}
+	if len(args) != 1 {
+		return fmt.Errorf("exactly one NAME is required, got %d", len(args))
+	}
+	kubeconfig, err := raw.ResolveKubeconfig(cmd)
+	if err != nil {
+		return err
+	}
+	restconfigTmpl := rest.CopyConfig(kubeconfig)
+	raw.SetupKubeconfig(restconfigTmpl)
+	k8sClient, err := client.New(kubeconfig, client.Options{Scheme: multiclusterscheme.Scheme})
+	if err != nil {
+		return err
+	}
+
+	serviceAccount := corev1.ServiceAccount{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      accessTokenOpts.serviceAccount,
+			Namespace: accessTokenOpts.namespace,
+		},
+	}
+
+	fmt.Fprintln(cmd.OutOrStdout(), fmt.Sprintf(`Creating ServiceAccount "%s"`, accessTokenOpts.serviceAccount))
+	if err := k8sClient.Create(context.TODO(), &serviceAccount); err != nil {
+		if errors.IsAlreadyExists(err) {
+			fmt.Fprintln(cmd.OutOrStderr(), fmt.Sprintf(`The ServiceAccount "%s" already exists`, accessTokenOpts.serviceAccount))
```

Comment:
Perhaps `fmt.Fprintf` is more suitable here if formating is required:
```
fmt.Fprintf(cmd.OutOrStderr(), `The ServiceAccount "%s" already exists\n`, accessTokenOpts.serviceAccount)
```

Reply:
I used raw string in output messages. So the escape character "\" won't work if use "fmt.Fprintf".  The output will be "The ServiceAccount "xyz" already exists\n". That's why I changed all the "fmt.Fprintf" to "fmt.Fprintln"

Reply:
```
fmt.Fprintf(cmd.OutOrStderr(), "The ServiceAccount \"%s\" already exists\n", accessTokenOpts.serviceAccount)
```

Reply:
Sure. I changed all the `fmt.Fprintln(....fmt.Sprintf)` to `fmt.Fprintf`